game-core/src/test/java/games/strategy/triplea/delegate/battle/MustFightBattleExecutablesTest.java

```diff
@@ -499,47 +495,6 @@ private void assertThatFirstStrikeStepOrder(
     assertThat(indices.get(stepOrder.get(1)), lessThan(indices.get(stepOrder.get(2))));
   }
 
-  private void assertThatFirstStrikeReturnFireIs(
-      final MustFightBattle battle,
-      final MustFightBattle.ReturnFire returnFire,
-      final boolean attacker) {
-    final List<IExecutable> execs = battle.getBattleExecutables();
-    final int index =
-        getIndex(
-            execs,
-            attacker
-                ? MustFightBattle.FirstStrikeAttackersFire.class
-                : MustFightBattle.FirstStrikeDefendersFire.class);
-    final IExecutable step = execs.get(index);
-
-    final IDelegateBridge delegateBridge = mock(IDelegateBridge.class);
-    step.execute(null, delegateBridge);
-
-    if (attacker) {
-      verify(battle).firstStrikeAttackersFire(returnFire);
-    } else {
-      verify(battle).firstStrikeDefendersFire(returnFire);
-    }
-  }
-
-  @Test
-  @DisplayName(
-      "When attacker has a destroyer, defender has a destroyer, WW2v2 is true, "
-          + "and DEFENDING_SUBS_SNEAK_ATTACK is either, then attacker has return fire all")
-  void firstStrikeAttackerReturnFireAttHasDestroyerDefHasDestroyerWW2v2TrueSneakAttackTrueFalse() {
```

Comment:
All of these tests are replaced with the tests in the three FirstStrike step classes.